spring-cloud-gcp-autoconfigure/src/test/java/org/springframework/cloud/gcp/autoconfigure/datastore/it/GcpDatastoreEmulatorIntegrationTests.java

```diff
@@ -57,6 +60,14 @@
  */
 public class GcpDatastoreEmulatorIntegrationTests {
 
+	@BeforeClass
+	public static void checkToRun() {
```

Comment:
Yeah, I think it's a good idea to mark emulator tests as being integration tests once and for all.

Reply:
Ack.